imagefiles/dockcross

```diff
@@ -185,11 +185,13 @@ fi
 if [ -n "$UBUNTU_ON_WINDOWS" ]; then
     WSL_ROOT="/mnt/"
     CFG_FILE=/etc/wsl.conf
-    CFG_CONTENT=$(cat $CFG_FILE | sed -r '/[^=]+=[^=]+/!d' | sed -r 's/\s+=\s/=/g')
-    eval "$CFG_CONTENT"
-    if [ -n "$root" ]; then
-        WSL_ROOT=$root
-    fi
+	if [ -f CFG_FILE ]; then
```

Comment:
I think the `$` is missing. The following may be the correct syntax: `if [ -f $CFG_FILE ]; then`